lib/ftu/homepage.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import './authentication.dart';
+import 'package:firebase_database/firebase_database.dart';
+//import './todo.dart';
+import 'dart:async';
+
+class HomePage extends StatefulWidget {
+  HomePage({Key key, this.auth, this.userId, this.logoutCallback})
+      : super(key: key);
+
+  final BaseAuth auth;
+  final VoidCallback logoutCallback;
+  final String userId;
+
+  @override
+  State<StatefulWidget> createState() => new _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  
+
+  final FirebaseDatabase _database = FirebaseDatabase.instance;
+  final GlobalKey<FormState> formKey = GlobalKey<FormState>();
+
+  @override
+  Widget build(BuildContext context) {
+    return new Scaffold(
+      appBar: new AppBar(
+        title: new Text("HomePage"),
+      ),
+      body: new Container(
+        child: new Text("Hello World"),
```

Comment:
Should make a constants file and import this text and the text on L29 from that constants file

Reply:
this still needs fixing